src/libraries/Hooks.sol

```diff
@@ -117,7 +117,7 @@ library Hooks {
         // If a hook contract is set, it must have at least 1 flag set, or have a dynamic fee
         return address(self) == address(0)
             ? !fee.isDynamicFee()
-            : (uint160(address(self)) >= AFTER_REMOVE_LIQUIDITY_RETURNS_DELTA_FLAG || fee.isDynamicFee());
+            : (uint160(address(self)) & uint160((1 << 14) - 1) > 0 || fee.isDynamicFee());
```

Comment:
might be nice to make a new constant `ALL_HOOK_MASK = uint160((1 << 14) - 1))` or something to put here